templates/default/mysql-job.sh.erb

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+
+# rdiff-backup MySQL job script for $JOB managed by Chef. Changes will be overwritten.
+
+# This script is normally run by cron, but can be run manually to start the job if it is not already running.
+
+FILE=`basename $0`
+DIR=`dirname $0`
+PIDFILE="$DIR/.$FILE.lock"
+JOB="${FILE%.sh}"
+LOGPATH='/var/log/rdiff-backup'
+INSTANCES=`pgrep -fu <%= @suser %> $FILE | wc -l`
+
+# Prepends date to every line of stdin, then writes to both stdout and the log file.
```

Comment:
Using the word both to describe what the function that doesn't have the word both in its name is probably not the best way to describe it.